### PR TITLE
Croatian translation

### PR DESCRIPTION
**Pull request by arvvoid**
### Check if this PR fulfills these requirements:
* [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [X] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [X] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [X] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [X] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
Update to Croatian translation

### CODE REVIEW

**Comment by arvvoid** [comment]
Work in progress not ready for merging

File: resources/language/resource.language.hr_hr/strings.po [hunk 1]
```diff
@@ -518,7 +518,10 @@ msgid "Sync My List to Kodi library"
 msgstr "Sinkroniziraj Moju Listu sa Kodi zbirkom"
 
 msgctxt "#30123"
-msgid "Do you really want to proceed?\nThis will delete all previously exported items and export all items from My List for the currently active profile.\nThis operation may take some time..."
+msgid ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.hr_hr/strings.po]: do not change structure of msgid

Comment (arvvoid) [reply resources/language/resource.language.hr_hr/strings.po]: Will fix that PoEdit did this automatically when saving the file...

Comment (arvvoid) [reply resources/language/resource.language.hr_hr/strings.po]: This should be fine now

File: resources/language/resource.language.hr_hr/strings.po [hunk 2]
```diff
@@ -931,40 +919,49 @@ msgstr "Dječji račun"
...
+msgstr "Ograniči prema razini zrelosti"
+
+msgctxt "#30233"
+msgid "Content for {} will require the PIN to start playback."
+msgstr "Za sadržaj {} bit će potreban PIN da bi se započela reprodukcija."
+
+msgctxt "#30234"
+msgid "Keep watched status marks separate for each profile (this option will be removed in future versions)"
+msgstr "Oznake statusa gledanja držite odvojeno za svaki profil (ova će opcija biti uklonjena u
+budućim verzijama)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hr_hr/strings.po]: all good, only this minor fix to do

Comment (arvvoid) [reply resources/language/resource.language.hr_hr/strings.po]: Done

**Comment by arvvoid** [comment]
It should be good to go now :)

**Comment by CastagnaIT** [comment]
perfect thanks

**Pull request merged**